sugar_bench_test.go

```diff
@@ -0,0 +1,54 @@
+// Copyright (c) 2016 Uber Technologies, Inc.
```

Comment:
All your points about benchmarks are absolutely valid. But my intention for this file was to have means of comparison between core logger and sugared one. So I basically mirrored `Benchmark10Fields` from `logger_test_bench.go`.